src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -28,6 +29,25 @@ import StepPropTypes from './StepPropTypes';
 
 const propTypes = {
     ...StepPropTypes,
+
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }),
+
+    /** Object with various information about the user */
+    user: PropTypes.shape({
```

Comment:
Same as here detected used with `shape` type.

Reply:
Is it a bug or an expected result?

Reply:
Yeah, it seems like a bug from `eslint-plugin-react`. Okay, this component is passed!